tests/images.spec.ts

```diff
@@ -18,3 +24,47 @@ test("search for custom image and create an instance from it", async ({
 
   await deleteInstance(page, customInstance);
 });
+
+test("Export and Upload an image", async ({ page }) => {
+  // creating an image from an instance, because we can only export unified images
+  // and can't test the export with a standard image fetched from the image server
+  // we can remove this step once the export of split images is enabled by the backend.
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  let imageAlias = await createImageFromInstance(page, instance);
+  const downloadPromise = page.waitForEvent("download");
+
+  await page.getByRole("link", { name: "Images", exact: true }).click();
+  await page.getByPlaceholder("Search").click();
+  await page.getByPlaceholder("Search").fill(imageAlias);
+
+  await expect(page.getByText(imageAlias)).toBeVisible();
+  let imageName = await getImageNameFromAlias(page, imageAlias);
+  await page.getByLabel("export image").click();
+  const download = await downloadPromise;
+  await page.waitForSelector(
+    `text=Image ${imageName} was successfully downloaded. Please check your downloads folder.`,
+  );
+  const IMAGE_FILE = "tests/fixtures/image.tar.gz";
+  await download.saveAs(IMAGE_FILE);
+  await deleteImage(page, imageName);
+
+  //Upload an image
+  await page.goto("/ui/");
+  await page.getByRole("link", { name: "Images", exact: true }).click();
+
+  await page.getByRole("button", { name: "Upload image" }).click();
+  await page.getByLabel("Image backup file").setInputFiles(IMAGE_FILE);
```

Comment:
```suggestion
  await page.getByLabel("Image backup file").setInputFiles(download.path());
```
I think we can directly use the download path, without saving it manually.

Reply:
So apparently the download path cannot be used because it only works when playwright is used locally, not remotely, as it is being done when we run tests. I read this online somewhere but download.path is not an option for us and to circumvent it, we should use a static path.